recipe/meta.yaml

```diff
@@ -16,20 +16,20 @@ build:
   features:
```

Comment:
Would add the following as well and re-render. The reason being Python 3.5 and 3.6 both use the same `vc` version. So we only need to build one of them. Easiest being Python 3.6 as we will likely drop Python 3.5 in the near future (when Python 3.7 comes out).

```yaml
  skip: true  # [win and py35]
```

Reply:
Done above.